docs/Custom_Operation_for_GPUs.md

```diff
@@ -34,7 +34,7 @@ You need to follow these steps in Python:
 * Define its abstract evaluation.
 * Define its lowering to MLIR.
 * [Optional] Define the gradient.
-* [Optional] Use [xmap](https://jax.readthedocs.io/en/latest/notebooks/xmap_tutorial.html) (or one of the experimental [custom_partitioning](https://jax.readthedocs.io/en/latest/jax.experimental.custom_partitioning.html) or [shard_map](https://jax.readthedocs.io/en/latest/jep/14273-shard-map.html) functions) for fast multi-GPU.
+* [Optional] Use [xmap](https://jax.readthedocs.io/en/latest/notebooks/xmap_tutorial.html), [custom_partitioning](https://jax.readthedocs.io/en/latest/jax.experimental.custom_partitioning.html) or [shard_map](https://jax.readthedocs.io/en/latest/jep/14273-shard-map.html) functions for fast multi-GPU.
```

Comment:
Move xmap to the end and say it is deprecated?

Reply:
It is mentioned a little later in the doc that xmap is deprecated. I will just remove it from here.